source/index.html.erb

```diff
@@ -143,13 +143,23 @@
     </div>
     <div class="content-section content-section--with-top-border">
       <div class="grid-row">
-          <div class="column-full">
-            <h2 class="content-section__title heading-medium">GOV.UK&nbsp;Pay is already working with:</h2>
-            <img class="content-section__image partner-logo" src="/pay-product-page/images/ministry-of-justice.png" alt="Logo of the Ministry of Justice" />
-            <img class="content-section__image partner-logo" src="/pay-product-page/images/home-office.png" alt="Logo of the Home Ofiice" />
-            <img class="content-section__image partner-logo" src="/pay-product-page/images/beis.png" alt="Logo of the Department for Business, Energy &amp; Industrial Strategy" />
-            <img class="content-section__image partner-logo" src="/pay-product-page/images/office-public-guardian.png" alt="Logo of the Office of the Public Guardian" />
+        <div class="column-full">
+          <h2 class="content-section__title heading-medium">GOV.UK&nbsp;Pay is already working with:</h2>
+          <div class="cf">
+            <img class="content-section__image partner-logo" src="/pay-product-page/images/logos/moj.png" alt="Logo of the Ministry of Justice" />
+            <img class="content-section__image partner-logo" src="/pay-product-page/images/logos/dit.png" alt="Logo of the Department of International Trade" />
+            <img class="content-section__image partner-logo" src="/pay-product-page/images/logos/home-office.png" alt="Logo of the Home Ofiice" />
+            <img class="content-section__image partner-logo" src="/pay-product-page/images/logos/fco.png" alt="Logo of the Foreign and Commonwealth Office" />
+            <img class="content-section__image partner-logo" src="/pay-product-page/images/logos/mod.png" alt="Logo of the Ministry of Defence" />
           </div>
+          <div class="cf">
+            <img class="content-section__image partner-logo" src="/pay-product-page/images/logos/ccc.png" alt="Logo of Canterbury City Council" />
+            <img class="content-section__image partner-logo partner-logo-smaller" src="/pay-product-page/images/logos/kcc.png" alt="Logo of Kent County Council" />
+            <img class="content-section__image partner-logo" src="/pay-product-page/images/logos/disclosure-scotland.png" alt="Logo of Disclosure Scotland" />
+            <img class="content-section__image partner-logo" src="/pay-product-page/images/logos/stratford.png" alt="Logo of Stratford-upon-Avon District Council" />
+            <img class="content-section__image partner-logo" src="/pay-product-page/images/logos/nhs.png" alt="Logo of National Health Service" />
```

Comment:
“NHS” is more well-known than “National Health Service” and matches the text in the logo